internal/introspect/annotator.go

```diff
@@ -83,3 +84,7 @@ func (a *Annotator) SetDefault(i any, defaultValue any, env ...string) {
 	a.Defaults[field.Name] = defaultValue
 	a.DefaultEnvs[field.Name] = append(a.DefaultEnvs[field.Name], env...)
 }
+
+func (a *Annotator) SetToken(module, token string) {
+	a.Token = fmt.Sprintf("pkg:%s:%s", module, token)
```

Comment:
Should `SetToken` test the passed parameters in any way? For the `module` I'd check for traling/leading `:` and whitespaces. For the `module` I'd check at least for leading `:` and whitespaces. For the result of `fmt.Sprintf` I'd call `TrimSpace` to ensure for no trainglin whitespaces.

Reply:
We can add that.

Reply:
Like it. Makes the function more reliable. 👍🏼